src/fsharp/FSharp.Compiler/FSharp.Compiler.fsproj

```diff
@@ -456,6 +462,21 @@
     <Compile Include="..\fsc.fs">
       <Link>Driver\fsc.fs</Link>
     </Compile>
+    <!--
```

Comment:
Please remove this comment - it's only understandable in the context of the specific "no duplication" aims being discussed in this PR. thanks